modules/core/02-client/keeper/keeper.go

```diff
@@ -403,3 +403,12 @@ func (k Keeper) ClientStore(ctx sdk.Context, clientID string) sdk.KVStore {
 	clientPrefix := []byte(fmt.Sprintf("%s/%s/", host.KeyClientStorePrefix, clientID))
 	return prefix.NewStore(ctx.KVStore(k.storeKey), clientPrefix)
 }
+
+// GetClientStatus returns the status for a given clientState. If the client type is not in the allowed
+// clients param field, Unauthorized is returned, otherwise the client state status is returned.
+func (k Keeper) GetClientStatus(ctx sdk.Context, clientState exported.ClientState, clientID string) exported.Status {
```

Comment:
Should we add documentation somewhere that removing clients from the `AllowedClients` parameter will disable all usage of that client type?

Reply:
Yes I think we can improve this docstring a bit!